modules/monitoring-services.nix

```diff
@@ -545,7 +579,10 @@ in {
         prometheus = {
           enable = true;
           webExternalUrl = "https://${cfg.webhost}/prometheus/";
-          extraFlags = [ "--storage.tsdb.retention=8760h" ];
+          extraFlags = [
+            "--storage.tsdb.retention=${cfg.prometheus.storageRetentionTime}"
```

Comment:
`storage.tsdb.retention` is now deprecated in favor of 
`storage.tsdb.retention.time` since 2.8 (and we should be on 2.13)

Reply:
Ah, thanks